src/promptflow/promptflow/azure/_storage/cosmosdb/span.py

```diff
@@ -38,6 +39,7 @@ def __init__(self, span: SpanEntity, created_by: dict) -> None:
         self.links = span._content[SpanFieldName.LINKS]
         self.resource = span._content[SpanFieldName.RESOURCE]
         self.partition_key = span.session_id
```

Comment:
Do we also change the "partition_key" to collection_id in this PR?

Reply:
No, that's another topic. We even not yet decide use `collection_id` as partition key.